garrysmod/resource/OptionsSubAudio.res

```diff
@@ -471,7 +471,7 @@
 		"visible"		"1"
 		"enabled"		"1"
 		"tabPosition"		"0"
-		"labelText"		"Sound effect volume"
+		"labelText"		"#GameUI_SoundEffectVolume"
```

Comment:
You cannot not change existing localization strings.

`#GameUI_SoundEffectVolume` should remain where it was, and a new string can be added for `Sound effect volume` (for example `"#GameUI_SoundFXVolume"`)